WcaOnRails/spec/i18n_locale_files_spec.rb

```diff
@@ -14,3 +14,14 @@
     end
   end
 end
+
+describe "Momentjs activation" do
+  let(:file_content) { File.read(Rails.root.join('app', 'assets', 'javascripts', 'application.js')) }
+  let(:locales) { I18n.available_locales - [:en] }
+
+  (I18n.available_locales - [:en]).each do |locale|
```

Comment:
`I18n.available_locales - [:en]` is used twice, so what about making a variable like `other_locales` / whatever.